### PR TITLE
[DNM] Adds towels to marines that wake from cryosleep

### PR DESCRIPTION
**Pull request by Iatots**
Whoops! Can't have those bits show in my christian server!
Marines that wake from cryo now have towels to hide their private bits. 

Significantly adds gameplay value to maint techs, who will now have to fetch these once marines drop them.
Maid costume and washing machine in a far future PR.

Edit: I have found some good shit on polaris/virgo, am considering tinkering with and porting the result over. Might also make another PR since this one has 30 commits.

### CODE REVIEW

**Comment by FantasticFwoosh** [comment]
Yeah show those baldies who's the boss of this gym.

File: code/game/objects/items/weapons/twohanded.dm [hunk 1]
```diff
@@ -248,4 +248,40 @@
...
+	flags_equip_slot = SLOT_OCLOTHING
+	flags_atom = FPRINT|CONDUCT //it's moist
+	flags_item = TWOHANDED|NOBLUDGEON
+	force_wielded = 2
+	attack_verb = list("whomped", "smacked", "slapped", "schaffed")
+	
+/obj/item/weapon/twohanded/towel/wield(mob/user)
+	. = ..()
+	if(!.) return
+	icon_state += "_w"
```
[the remark this answers is not in the document]
Comment (ninjanomnom) [reply code/game/objects/items/weapons/twohanded.dm]: You could but it isn't necessary in this pr. We'll be implementing better code standards from here on in but the existing code needs to be updated piecemeal.

File: code/game/objects/items/weapons/twohanded.dm [hunk 2]
```diff
@@ -248,4 +248,40 @@
...
+ 
+/obj/item/weapon/twohanded/towel
+	name = "towel"
+	desc = "A cheap piece of cloth, dank with cryo juices. Can be wrung to free some moisture."
+	icon_state = "towel"
+	item_state = "towel"
+	damtype = HALLOSS
+	force = 0
+	sharp = 0
+	edge = 0
```
[the remark this answers is not in the document]
Comment (Iatots) [reply code/game/objects/items/weapons/twohanded.dm]: Wait, weapons with edge have a 1. Why are we mixing 0/1 and true/false?

[the remark this answers is not in the document]
Comment (Iatots) [reply code/game/objects/items/weapons/twohanded.dm]: https://github.com/ColonialMarines-Mirror/ColonialMarines-2018/blob/4b9ee09dc413b5f691b74ddfb6373b75abe57768/code/game/objects/items.dm#L19

**Pull request closed without merging**